### ISSUE TITLE
Include affiliated package contributors in team page

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy.github.com#63]
This page should include a list of affiliated package contributors (we need to decide whether to duplicate names already in the list of core contributors):

http://www.astropy.org/about.html

**Comment by eteq** [comment]
Hmm... I'm not sure how we would go about doing this, @astrofrog.  A manual list I think would quickly spiral out of control and not be sustainable.  And because not all of the affiliated package follow the same docs format, I'm not sure how to automate scraping the list from the various doc pages...

Perhaps instead a set of links to all the affiliated package's author list pages? That's a bit more indirect, but would be more manageable to keep up-to-date...

**Comment by embray** [comment]
The current list includes even people who have made like one typo fix in the docs and have never since contributed again.  But I'm okay with that and I believe in crediting them.  So I'm not too worried about spiraling out of control so long as affiliated package authors take it upon themselves to credit their contributors here.  We could encourage that as part of being an affiliated package.

**Comment by eteq** [comment]
I'm ok with your suggestion @astrofrog - I'm concerned it will be either annoying to manage or highly incomplete, but neither of those are the end of the world.

I would say to make it a bit easier we should have a clear policy along the lines of "any PR editing that list has to be accompanied by a statement from an affiliated package author that it's a real contributor".  That way there's at least some standard rather than just any PR gets accepted.

**Comment by astrofrog** [comment]
We now do this for the coordinated packages: https://www.astropy.org/team.html

**Comment by bsipocz** [comment]
We do have the maintainers listed, but not the contributors. It would be quite the task to merge all the lists (updating that list is already one of the pain points for doing an astropy release).

**Comment by astrofrog** [comment]
Oh yes oops I misunderstood.

**Comment by astrofrog** [comment]
Coming back to this, what we need is to include anyone who has contributed to a repository in the astropy org (so core, infrastructure, educational, coordinate, but *not* affiliated packages). We can probably re-use the mailmap file from the core package.

**Comment by pllim** [comment]
I think we have this now?

**Comment by bsipocz** [comment]
> I think we have this now?

Not exactly, the team page only lists core package contributors. I would strongly recommend to include those contributing to coordinated and infrastructure package, too.


(a lot of the recently closed affiliated-related don't fix issues could in fact stay open and rephrased for coordinated packages. E.g. dropdown/cross searchable documentation within the core and coordinated packages, so users can find reproject/regions/photutils functionality from the main astropy website. )

**Comment by pllim** [comment]
The state of Coordinated isn't shiny either. Sorry if I closed the issues prematurely. Feel free to re-open them or ping me in specific ones you want to be re-opened.
